chain/testing.go

```diff
@@ -21,13 +22,17 @@ import (
 // State root CIDs are computed by an abstract StateBuilder. The default FakeStateBuilder produces
 // state CIDs that are distinct but not CIDs of any real state tree. A more sophisticated
 // builder could actually apply the messages to a state tree (not yet implemented).
+// The builder is deterministic: two builders receiving the same sequence of calls will produce
+// exactly the same chain.
```

Comment:
I know it is implied, but do you think it's worth adding a note stating that the miner address passed to `NewBuilder` is basically a "seed" for the chain it generates? I tripped over this bit the first time I used it.

Reply:
Actually I didn't think of it that way. We may want to add an explicit seed later.

In fact I don't think this miner address will last because it causes the chains to be "wrong" in that multiple blocks at the same height have the same miner. I expect to move to random miner addresses.